usr/src/test/test-runner/cmd/run.py

```diff
@@ -144,13 +145,16 @@ class Cmd(object):
     def __init__(self, pathname, outputdir=None, timeout=None, user=None):
         self.pathname = pathname
         self.outputdir = outputdir or 'BASEDIR'
-        self.timeout = timeout or 60
+        self.timeout = timeout
         self.user = user or ''
         self.killed = False
         self.result = Result()
 
+        if self.timeout == None:
```

Comment:
Normally, I'd agree, but if we use `if not self.timeout` then when timeout is 0 (meaning infinite) we'll reset it to 60, which we don't want.